src/app/delegation_compliance/README.md

```diff
@@ -0,0 +1,30 @@
+# Delegation compliance
```

Comment:
Can you add a note here of exactly how you recommend running this against archive data (would you import a dump of the data locally and connect via there? Would you use kubectl to get a connection to the live database?)

Reply:
Added a mention.